external/stdk/patch/0001-patch-for-1.0.9_fix-overwrite-issue-return-value-and.patch

```diff
@@ -0,0 +1,58 @@
+From 03c7589cf27390f13f8270675c28a6b90b95bb76 Mon Sep 17 00:00:00 2001
+From: "qin.li" <qin.li@samsung.com>
+Date: Tue, 17 Dec 2019 16:32:50 +0800
+Subject: [PATCH] patch for 1.0.9_fix overwrite issue, return value, and remove
+ OS config for ssl
+
+---
+ src/easysetup/iot_easysetup_d2d.c      | 4 ++--
+ src/port/net/openssl/iot_net_openssl.c | 4 ++--
+ 2 files changed, 4 insertions(+), 4 deletions(-)
+
+diff --git a/src/easysetup/iot_easysetup_d2d.c b/src/easysetup/iot_easysetup_d2d.c
+index 2d55f60..79ce1b3 100644
+--- a/src/easysetup/iot_easysetup_d2d.c
++++ b/src/easysetup/iot_easysetup_d2d.c
+@@ -444,7 +444,7 @@ static iot_error_t _es_keyinfo_handler(struct iot_context *ctx, char *in_payload
+ 		goto exit;
+ 	}
+ 
+-	for (i = 0, j = 0; i < sizeof(rand_asc); i += 2, j++) {
++	for (i = 0, j = 0; i < sizeof(rand_asc) - 1; i += 2, j++) {
```

Comment:
let's sync up with 1.0.10,     then this patch is not needed anymore

Reply:
Good idea, working on it.